### PR TITLE
feat(storybook): add component DocNavigator

### PR DESCRIPTION
**Pull request by astagnol**
DocNavigator displays in Documentation a button to the Demo page and a link to the Zeroheight specifications

### CODE REVIEW

File: packages/storybook/package.json [hunk 1]
```diff
@@ -21,6 +21,7 @@
   "devDependencies": {
     "@storybook/addon-a11y": "8.0.4",
     "@storybook/addon-essentials": "8.0.4",
+    "@storybook/addon-links": "^8.0.8",
```
Comment (dpellier) [inline packages/storybook/package.json:24]: never use "^" and set the same version as other
`"@storybook/addon-links": "8.0.4",`

File: packages/storybook/stories/doc-navigator.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      <div style={{ marginLeft: '20px' }}>
+        <OdsLink
+          label={'Go to Zeroheight'}
+          icon={'external-link'}
+          onClick={() => {
+            const storyId = demoStoryKeys[0];
+            const linkId = extractLinkId(storyId);
+
+            if (linkId) {
+              const baseUrl = 'https://zeroheight.com/6fc8a63f7/p/';
```
Comment (dpellier) [inline packages/storybook/stories/doc-navigator.tsx:47]: could be moved to the same place as the link ids enum

**Review by dpellier — CHANGES_REQUESTED** [review]

File: packages/storybook/stories/doc-navigator.tsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+            onClick={() => {
+              navigate({ storyId })
+            }}
+          >
+          </OdsButton>
+        ))}
+      </div>
+      <div style={{ marginLeft: '20px' }}>
+        <OdsLink
+          label={'Go to Zeroheight'}
```
Comment (dpellier) [inline packages/storybook/stories/doc-navigator.tsx:40]: Maybe a less tool-oriented label? (Go to functional specification?)
BTW for fixed string you can omit the `{}` (`label="label"`)

File: packages/storybook/stories/doc-navigator.tsx [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+  };
+
+  return (
+    <div style={{ display: 'flex', justifyContent: 'center', alignItems: 'center' }}>
+      <div>
+        {demoStoryKeys.map((storyId, index) => (
+          <OdsButton
+            key={index}
+            label={'Go to Demo'}
+            variant={'default'}
```
Comment (aesteves60) [inline packages/storybook/stories/doc-navigator.tsx:30]: Use the enum

File: packages/storybook/stories/doc-navigator.tsx [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+              navigate({ storyId })
+            }}
+          >
+          </OdsButton>
+        ))}
+      </div>
+      <div style={{ marginLeft: '20px' }}>
+        <OdsLink
+          label={'Go to Zeroheight'}
+          icon={'external-link'}
```
Comment (aesteves60) [inline packages/storybook/stories/doc-navigator.tsx:41]: use the enum ODS_ICON_NAME

File: packages/storybook/stories/zeroheight-link-ids.ts [hunk 6]
```diff
@@ -0,0 +1,42 @@
+export enum ZeroheightLinkIds {
```
Comment (aesteves60) [inline packages/storybook/stories/zeroheight-link-ids.ts:1]: export should be at the end of the file

**Review by dpellier — APPROVED** [review]

**Review by aesteves60 — APPROVED** [review]

**Pull request merged**